File: BEE 1005.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var a = parseFloat(lines.shift());
var b = parseFloat(lines.shift());

var avg = (a*3.5 + b*7.5)/(3.5+7.5);

console.log("MEDIA =", avg.toFixed(5));

File: BEE 1005.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{

        Scanner input = new Scanner(System.in);

        double a, b, avg;

        a = input.nextDouble();
        b = input.nextDouble();

        avg = (a*3.5 + b*7.5)/(3.5+7.5);

        System.out.printf("MEDIA = %.5f\n", avg);
    }
}